src/functionalTest/resources/features/F-004/S-040.td.json

```diff
@@ -0,0 +1,69 @@
+{
+  "_guid_": "S-040",
+  "_extends_": "F-004_Test_Base_data",
+  "title": "must successfully upload document with binary content",
+  "specs": [
+    "contains the metadata for the document uploaded above"
+  ],
+  "request": {
+    "headers": {
+      "caseTypeId": "BEFTA_CASETYPE_2_2",
+      "jurisdictionId": "BEFTA_JURISDICTION_2"
+    },
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "classification",
+          "value": "PUBLIC"
+        },
+        {
+          "key": "roles",
+          "value": "caseworker"
+        },
+        {
+          "key": "files",
```

Comment:
this should be "file", not "files".

Reply:
This upload document endpoint support multiple documents. thats why its key is files by default.